File: 005 - Financial Computation/python/main.py
while True:
    cmd = input('> ')
    if cmd == 'exit':
        exit()
    cmd = int(cmd)
    if cmd < 12500:
        print('Income = {} pixel dust'.format(cmd))
        print('Tax = {} pixel dust'.format(str(cmd*0.32).split('.')[0]))
    else:
        print('Income = {} pixel dust'.format(cmd))
        print('Tax = {} pixel dust'.format(str(cmd*0.65).split('.')[0]))

File: 005 - Financial Computation/c/main.c
#include "stdio.h"
#include "string.h"
int main() {
    int running = 1;
    int cmd_int;
    while (running == 1) {
        char cmd[100];
        printf("> ");
        fgets(cmd, 20, stdin);
        if (strcmp(cmd, "exit\n") == 0) {
            running = 0;
        } else {
            cmd_int = atoi(cmd);
            printf("Income = ");
            printf("%d", cmd_int);
            printf(" pixel dust\n");
            printf("Tax = ");
            if (cmd_int < 12500) {
                printf("%f", cmd_int*0.32);
            } else {
                printf("%f", cmd_int*0.65);
            }
            printf(" pixel dust\n");
        }
    }
}